zh_tw/news/_posts/2019-04-23-move-to-git-from-svn.md

```diff
@@ -15,7 +15,7 @@ lang: zh_tw
 
 * 在 cgit 上，我們不使用 Topic 分支。
 * GitHub 存放區將持續保持鏡像，我們不使用「Merge pull request」功能。
-* ruby_2_4、ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
+* ruby_2_4，ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
```

Comment:
```suggestion
* ruby_2_4、ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
```

Reply:
Sorry, it's my bad. Looks like I push an invisible character before the `、` again.
Like you found in #1983 last time.

But it's a little bit different between `、` and `，` in Chinese.
Could you just delete the invisible character and keep the `、`? Thanks!!